DirectEmployers/MyJobs
# [PD-3423] create testing for header reducer

## What's this PR do?
PR runs the test specs for the header reducer inside of the analytics application

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start by looking at the code or by running dkgg watch command inside of the myjobs folder

## How should this be manually tested?
Pull down the repository. Inside of the myjobs folder run dkgg watch and run the test to make sure they are all passing for the specs folders. Also you can look at the code.

## Any background context you want to provide?
